python/lsst/obs/base/defineVisits.py

```diff
@@ -279,6 +279,18 @@ class DefineVisitsConfig(Config):
         optional=False,
         default=True,
     )
+    update_records = Field(
```

Comment:
Aren't we going to want to surface these in the command-line tooling for ingest-raw and define-visit? Does `butler ingest-raw` override the config flag? Is this really a config option or is it a parameter for `run`?